src/primaite/agents/aegis/data.py

```diff
@@ -9,9 +9,12 @@ class GLLMDataset(Dataset):
     def __init__(self, graphs, questions, gt_answers, llm: LLM):
         self.graphs = graphs
         self.questions = questions
-        self.gt_answers = gt_answers
+        self.gt_answers = [
+            answer + " " + llm.tokenizer.eos_token for answer in gt_answers
```

Comment:
Maybe remove the additional space " " as it is no longer needed.

Reply:
thakns